README.md

```diff
@@ -1,4 +1,4 @@
-Zcoin
+<img src="https://zcoin.io/storage/2017/12/ci_logo_monochrome.svg" width="500">
```

Comment:
@reubenyap do we have a better URL than this one?

Reply:
I think size is ok for me .

Reply:
What is the issue with the URL?

Reply:
@reubenyap the problems with this URL is it look like it is managed by CMS. So it is better to use the one that we uploaded to the web server by ourself.

Reply:
@ultimaweapon where can we find that URL?

Reply:
Do you mean the one that we uploaded by ourself? If yes, no I don't have.

Reply:
@reubenyap can we upload this image to our website with a proper location?